pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -814,3 +843,53 @@ def plot_channel_contributions_grid(
             ylabel="contribution",
         )
         return fig
+
+    def predict_posterior(
+        self,
+        X_pred: Union[np.ndarray, pd.DataFrame, pd.Series],
+        extend_idata: bool = True,
+        combined: bool = True,
+        include_last_observations: bool = False,
+        **kwargs,
+    ) -> DataArray:
```

Comment:
My interpretation and first pass from our conversation the other week 
@ricardoV94 @cetagostini 

The last_observations can optionally be used in the `predict_posterior` method. 
Begs the question, should this be supported the other predict-like methods, changing their parameters too? 
Would like to hear thoughts on this

Reply:
I think ideally yes to have consistent signatures.

Reply:
okay. Let me work towards that